execute/plugin.go

```diff
@@ -172,6 +166,9 @@ func (p *Plugin) Observation(
 		}
 	}
 
+	msgVisibilityInterval := p.cfg.MessageVisibilityInterval
+	fetchFrom := time.Now().Add(-msgVisibilityInterval)
```

Comment:
Can probably inline this into the call site, unless we're using for logging somewhere